packages/shared-components/src/CaseStudyQuestion.tsx

```
@@ -62,8 +81,27 @@ function CaseStudyQuestion({
           <div>
+            <div className="-mb-2 mt-4 flex w-full flex-row flex-wrap gap-4">
+              {options.cases.map((_, index) => {
+                return (
+                  <div
+                    onClick={() => setCaseIndex(index)}
+                    className={twMerge(
+                      'bg-uzh-grey-60 rounded-full px-2.5 py-0.5 text-sm font-bold hover:cursor-pointer',
+                      caseIndex === index && 'bg-uzh-blue-80 text-white',
+                      caseValidity[index] && 'bg-green-700 text-white'
+                    )}
+                  >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)